src/global_scheduler/global_scheduler.c

```diff
@@ -71,6 +72,24 @@ void process_new_db_client(db_client_id db_client_id,
   }
 }
 
+/* object table subscribe callback
```

Comment:
Can you bring this in the same format as the other format strings?

first line: /**
second line: * with a one line description of the fct
third line: emtpy
then the args
then @return Void.